src/adhocracy_core/adhocracy_core/sheets/polarization.py

```diff
@@ -0,0 +1,47 @@
+"""Polarization sheet."""
+import colander
+
+from adhocracy_core.sheets import sheet_meta
+from adhocracy_core.interfaces import IPredicateSheet
+from adhocracy_core.sheets import add_sheet_to_registry
+from adhocracy_core.schema import AdhocracySchemaNode
+
+
+class IPolarizable(IPredicateSheet):
+
+    """Marker interface for the polarization sheet."""
+
+
+class Position(AdhocracySchemaNode):
+
+    """Schema node for the side (pro or contra)."""
+
+    schema_type = colander.String
+    missing = colander.drop
+    default = 'pro'
+    validator = colander.OneOf(['pro', 'contra']),
+
+
+class PolarizableSchema(colander.MappingSchema):
+
+    """
+    Polarizable sheet data structure.
+
+    `position`: the position in the debate, 'pro' or 'contra'.
+    """
+
+    position = Position()
```

Comment:
IPredicateSheet sheets have a "predicate" name like like, vote, and the reference fields subject and object.
This is missing here.